appservice/src/deploy/deploy.ts

```diff
@@ -68,20 +69,30 @@ export async function deploy(client: SiteClient, fsPath: string, configurationSe
                 break;
         }
 
-        const displayUrl: string = client.isFunctionApp ? '' : client.defaultHostName;
-        const deployComplete: string = displayUrl ?
-            localize('deployCompleteWithUrl', 'Deployment to "{0}" completed: {1}', client.fullName, `https://${displayUrl}`) :
+        const deployComplete: string = !client.isFunctionApp ?
+            localize('deployCompleteWithUrl', 'Deployment to "{0}" completed: {1}', client.fullName, client.defaultHostUrl) :
             localize('deployComplete', 'Deployment to "{0}" completed.', client.fullName);
         ext.outputChannel.appendLine(deployComplete);
         ext.outputChannel.appendLine('');
         const viewLogs: MessageItem = {
             title: localize('viewLogs', 'View Logs')
         };
+        const browseWebsite: MessageItem = {
+            title: localize('browseWebsite', 'Browse Website')
+        };
+
+        const buttons: MessageItem[] = [viewLogs];
+        if (!client.isFunctionApp) {
+            buttons.push(browseWebsite);
+        }
 
         // Note: intentionally not waiting for the result of this before returning
-        window.showInformationMessage(deployComplete, viewLogs).then((result: MessageItem | undefined) => {
+        window.showInformationMessage(deployComplete, ...buttons).then(async (result: MessageItem | undefined) => {
             if (result === viewLogs) {
                 ext.outputChannel.show();
+            } else if (result === browseWebsite) {
+                // tslint:disable-next-line:no-unsafe-any
+                opn(client.defaultHostUrl);
```

Comment:
Is it easy to use the treeItem's browse website function?  If not, don't worry about it.

Reply:
I don't have access to the treeItem from the shared package